src/ontology/edits/oeo-physical.omn

```diff
@@ -2973,6 +2980,18 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/411",
         OEO_00000011
     
     
+Class: OEO_00010023
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A vehicle is an artifical object that used for transporting people or goods.",
```

Comment:
There is an `is` missing in the definition

Reply:
Thanks for spotting. I already merged this PR but changed it in the `feature/vehicles-#425` now.